src/concrete/ml/sklearn/tree_to_numpy.py

```diff
@@ -252,7 +274,7 @@ def tree_onnx_graph_preprocessing(
         # sklearn models apply the reduce sum before the transpose.
         # To have equivalent output between xgboost in sklearn,
         # apply the transpose before returning the output.
-        add_transpose_after_last_node(onnx_model)
+        add_transpose_after_last_node(onnx_model, use_fhe_sum)
```

Comment:
this function is only used for trees.
So it won't alter the behavior.

Reply:
we are in the `tree_to_numpy` file (where this function only appears) so no worries yes